devito/functions/basic.py

```diff
@@ -16,7 +16,8 @@
 from devito.tools import (ArgProvider, EnrichedTuple, Pickable, ctypes_to_cstr,
                           dtype_to_cstr, dtype_to_ctype)
 
-__all__ = ['Symbol', 'Indexed']
+__all__ = ['Symbol', 'Indexed', 'AbstractSymbol', 'Scalar', 'Object', 'CompositeObject',
+           'Array', 'LocalObject', 'AbstractFunction', 'CacheManager', '_SymbolCache']
```

Comment:
you're right, but I'm just saying to drop some from the list above, as otherwise they might mess up with the backend infrastructure, since basically they need backend-specific imports